providers/openfeature-provider-flagd/src/openfeature/contrib/provider/flagd/resolvers/process/connector/grpc_watcher.py

```diff
@@ -0,0 +1,92 @@
+import json
+import logging
+import threading
+import time
+import typing
+
+import grpc
+
+from openfeature.event import ProviderEventDetails
+from openfeature.exception import ParseError
+from openfeature.provider.provider import AbstractProvider
+
+from ....config import Config
+from ....proto.flagd.sync.v1 import sync_pb2, sync_pb2_grpc
```

Comment:
For more than one level up, an absolute import is much easier to read and to update if necessary.

Reply:
I was getting some mypy errors when using absolute import, need to check how to fix those:
```
providers/openfeature-provider-flagd/src/openfeature/contrib/provider/flagd/resolvers/process/connector/grpc_watcher.py:10: error:
Cannot find implementation or library stub for module named
"openfeature.contrib.provider.flagd.proto.flagd.sync.v1"  [import-not-found]
    from openfeature.contrib.provider.flagd.proto.flagd.sync.v1 import (
    ^
```

Reply:
Couldn't figure this out in a simple way, seems related to using pre-commit at root of a "mono-repo" but I don't see any clear resources of how to structure it better. Maybe this is a follow-up issue?

Reply:
This [approach](https://github.com/pre-commit/pre-commit/issues/466#issuecomment-274282684) that may work as long as the number of packages remains manageable.